.claude/skills/pr-merge/merge.py

```
@@ -0,0 +1,1269 @@
+    if allowed_tools:
+        cmd += ["--allowedTools", allowed_tools]
+    if permission_mode:
+        cmd += ["--permission-mode", permission_mode]
+    if schema:
+        cmd += ["--output-format", "json", "--json-schema", schema]
+    cmd += ["-p", prompt]
+
+    env = dict(os.environ, CLAUDE_HISTORY_ROLE=role)
+    env.pop("CLAUDECODE", None)          # removed, not blanked: headless nesting
+    try:
+        result = subprocess.run(cmd, capture_output=True, text=True,
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)